### DISCUSSION TITLE
Preformatted text in the release notes doesn't wrap correctly

### DISCUSSION THREAD

**Discussion by Permik** [community/community#6263] [Other Feature Feedback, Questions, & Ideas] [answered]
As mentioned in the title.
In the old version any preformatted text would be wrapped if it overflowed.

You can see it currently at the time of posting here: https://github.com/rust-analyzer/rust-analyzer/releases/tag/nightly

I also took a recording of it here: http://sx.permik.xyz/macup/2021-10-13/01-54-56.mp4

**Comment by MylesBorins** [comment, marked as answer]
@Permik is it safe to assume that you are publishing releases with no content in the body and relying on the tag / commit meta data for the body?

edit: We've identified the source of the issue and engineering is working on a fix 🎉

**Reply by MylesBorins** [reply]
I forked the repo to make a reproducible error, thanks for reporting!

https://github.com/MylesBorins/rust-analyzer/releases/tag/new-release

**Reply by MylesBorins** [reply]
This should be fixed now!
